contracts/UmaConditionalTokensBinaryAdapter.sol

```diff
@@ -392,96 +252,86 @@ contract UmaConditionalTokensBinaryAdapter is Auth, ReentrancyGuard {
         return payouts;
     }
 
-    function getExpectedResolutionData(QuestionData storage questionData) internal view returns (int256) {
-        return
-            getOptimisticOracle()
-                .getRequest(address(this), identifier, questionData.requestTimestamp, questionData.ancillaryData)
-                .resolvedPrice;
-    }
-
     /// @notice Resolves a question
     /// @param questionID - The unique questionID of the question
     function reportPayouts(bytes32 questionID) external {
         QuestionData storage questionData = questions[questionID];
 
-        require(!questionData.resolved, "Adapter::getExpectedPayouts: questionID is already resolved");
-        require(
-            block.number > questionData.settled,
-            "Adapter::reportPayouts: Attempting to settle and reportPayouts in the same block"
-        );
+        require(!questionData.resolved, AdapterErrors.AlreadyResolved);
+        require(block.number > questionData.settled, AdapterErrors.SameBlockSettleReport);
```

Comment:
I wonder if we want to add an additional delay, like 100 blocks, maybe unecessary

Reply:
hm is the thought to have some delay between settling the question and resolving the market?